style.css

```diff
@@ -113,14 +210,16 @@ nav ul {
     right: 1em;
 }
 
-.text-with-icon svg {
-    position: absolute;
-    width: 1em;
-    fill: var(--gray-2);
+.icon-top {
```

Comment:
Let's use BEM naming from the class and rename to `text-with-icon--top`. This way we know we have a "top" modifier for the text-with-icon.